cmd/contour-operator.go

```diff
@@ -44,13 +44,17 @@ func init() {
 }
 
 var (
-	image                string
+	contourImage         string
+	envoyImage           string
 	metricsAddr          string
 	enableLeaderElection bool
 )
 
 func main() {
-	flag.StringVar(&image, "image", "docker.io/projectcontour/contour:latest", "The image used for the managed Contour.")
+	flag.StringVar(&contourImage, "contour-image", "docker.io/projectcontour/contour:main",
```

Comment:
So does this define the default for the operator? Meaning, if I create a new "contour" CRD, then this is what is used if not specified?

Reply:
@stevesloka that is correct.